ng2-components/ng2-activiti-tasklist/src/components/comment-list.component.html

```diff
@@ -3,7 +3,7 @@
     (rowClick)="selectComment($event)" *ngIf="hasComments()">
 
     <data-columns>
-        <data-column key="createdBy">
+        <data-column key="createdBy" title="Created By">
```

Comment:
Can you please move values to i18n (en.json) files to enable localisation support? It is `ng2-activiti-tasklist/src/i18n/en.json`. The title would be then declared like:

```
<data-column key="createdBy" title="{{ 'KEY' | translate }}">
```